index.html

```diff
@@ -150,10 +150,10 @@ <h2>Introduction</h2>
       specifications:</p>
 
       <ul>
-      <li><code><dfn data-cite="!HTML/multipage/system-state.html#navigator">Navigator</dfn></code> [[!HTML]]</li>
-      <li><a class="externalDFN">DOMHighResTimeStamp</a> [[!HIGHRES-TIME]]</li>
-      <li><a class="externalDFN">WindowAnimationTiming</a> [[!ANIMATION-TIMING]]</li>
-      <li><a class="externalDFN">PerformanceTiming</a> [[!NAVIGATION-TIMING]]</li>
+      <li><code><dfn data-cite="!HTML/system-state.html#navigator">Navigator</dfn></code> [[!HTML]]</li>
+      <li><dfn data-cite="!hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</dfn></code> [[!HIGHRES-TIME]]</li>
```

Comment:
only put a "/" on actual multipage specs (or else the reference resolves to "/" against the base - which would be "https://w3c.org/#dom-domhighrestimestamp"). So, this should just be the following (you can also drop all the "!" - ReSpec is reference-context aware now):
```JS
data-cite="hr-time-2#dom-domhighrestimestamp"
```